xbmc/filesystem/AddonsDirectory.cpp

```diff
@@ -118,6 +133,37 @@ static void SetUpdateAvailProperties(CFileItemList &items)
   }
 }
 
+
+// Creates categories for addon types, if we have any addons with that type.
+static void GenerateTypeListing(
+  const CURL& path,
+  const std::vector<TYPE>& types,
+  const VECADDONS& addons,
+  CFileItemList& items)
```

Comment:
This breaking of method parameters doesn't fit the rest of the coding style at all.